addons/web/static/src/core/pwa/pwa_service.js

```diff
@@ -53,13 +53,8 @@ const pwaService = {
         });
 
         function _getInstallationState(scope = state.startUrl) {
-            let value;
-            try {
-                value = JSON.parse(browser.localStorage.getItem("pwa.installationState"))[scope];
-            } catch {
-                value = "";
-            }
-            return value;
+            const ls = JSON.parse(browser.localStorage.getItem("pwa.installationState") || "{}");
+            return ls[scope] ?? "";
```

Comment:
```suggestion
            const installationState = browser.localStorage.getItem("pwa.installationState") 
            return installationState ? JSON.parse(installationState)[scope] : ""
```

Reply:
Seems better. Let me just check if we can return undefined, in case there is an installation state, but not for a scope

Reply:
Yes, it's fine. Thank you!